DesignSafe-CI/portal
# task/DES-1944: add Published Files tab to site search with link to publication

## Overview: ##
Add a new tab to Site Search that filters for only published files.

## PR Status: ##


## Related Jira tickets: ##

* [DES-1944](https://jira.tacc.utexas.edu/browse/DES-1944)

## Summary of Changes: ##

## Testing Steps: ##
1. search "ipynb" in site search
2. Check that the "view publication" link works in the search listing.

## UI Photos:
![image](https://user-images.githubusercontent.com/12601812/119175721-ce29b080-ba2f-11eb-9efb-614aa059092b.png)


## Notes: ##
